.github/workflows/ksql.yml

```diff
@@ -0,0 +1,37 @@
+name: ksql
+
+on:
+  merge_group:
+  workflow_dispatch: 
+
+jobs:
+  build:
+    if: github.repository_owner == 'Informatievlaanderen'
+    name: Deploy ksql statements
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+
+    - name: Execute ksql statements
+      shell: bash
+      run: |
+        pushd .ksql
+        for FILE in *.ksql; do 
+          echo Start execution of $FILE
+          contents=$(<$FILE)
+          contents=$(sed 's|"|\\\"|g' <<< $contents) # escape double quotes
+          contents=$(sed ':a; N; $!ba; s/\n/ /g' <<< $contents) # single line
+          echo Contents of file:
+          echo $contents
+          echo \nCurl command:
+          curl --silent -w "\n%{http_code}\n" --location 'https://pksqlc-n65nz.eu-west-1.aws.confluent.cloud:443/ksql' \
+            --header 'Accept: application/vnd.ksql.v1+json' \
+            --header 'Content-Type: application/json' \
+            --header 'Authorization: Basic Q0xUQjU0NkdBSkM0T1VPUzo4SVZUM2p5ZHU5T25pQUEwSGZvV2FyVzduS3QvUENoTHoyMHBGaUJweC9mUk4wYnFkV0g4MVBQOFN2eHBwSFg3' \
```

Comment:
Doesn't the auth value needs to come from a secret?

Reply:
ideally yes, didn't get around to it yet